frontend/workflows/projectCatalog/src/details/quick-links.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import type { clutch as IClutch } from "@clutch-sh/api";
+import {
+  Card,
+  Grid,
+  Link,
+  Popper,
+  PopperItem,
+  Tooltip,
+  TooltipContainer,
+  Typography,
+} from "@clutch-sh/core";
+
+// If only a single link, then no popper is necessary
+const QuickLink = ({ link, linkGroupName, linkGroupImage }) => (
+  <Grid item key={link.name}>
+    <Tooltip title={linkGroupName}>
+      <TooltipContainer>
+        <Link href={link.url}>
+          <img width="32px" height="32px" src={linkGroupImage} alt={link.name} />
+        </Link>
+      </TooltipContainer>
+    </Tooltip>
+  </Grid>
+);
+
+// Have a popper in the case of multiple links per group
+const QuickLinkGroup = ({ linkGroupName, linkGroupImage, links }) => {
+  const anchorRef = React.useRef(null);
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Grid item key={linkGroupName}>
+      <Tooltip title={linkGroupName}>
+        <TooltipContainer>
+          {/* eslint-disable */}
```

Comment:
just curious, why disable lint here?

Reply:
We get the following errors:
            error  Visible, non-interactive elements with click handlers must have at least one keyboard listener  jsx-a11y/click-events-have-key-events
            error  Non-interactive elements should not be assigned mouse or keyboard event listeners               jsx-a11y/no-noninteractive-element-interactions

I tried the solutions I found on stackoverflow about assigning role to button, and a few other things, and none of those worked.
There's many directions we can go here.
Maybe a better solution is to use a `button` or another component (IconButton?)? I'm not sure. Just thought I'd disable until we come up with a better solution.

Reply:
I can make it work if I wrap it in a `button` or `IconButton` but the problem is then I couldn't figure out how to fix the styling (it wouldn't override). When I do those things I get unwanted boxes and color.

Reply:
So I got it to work with setting the background for an img. One issue is that the "hand" icon doesn't appear in that case. I wonder if that is ok or if that is a suboptimal UX

Reply:
should be able to fix that with css `cursor: pointer`

Reply:
yeah I tried that. Also I tried using Link but the Link from Clutch requires an href which doesn't fit this use case.

Reply:
I think you'll want something like 
```jsx
          <button
            type="button"
            style={{ padding: 0, background: "transparent", border: "0", cursor: "pointer" }}
            ref={anchorRef}
            onClick={() => setOpen(true)}
          >
            <img width={ICON_SIZE} height={ICON_SIZE} src={linkGroupImage} alt={linkGroupName} />
          </button>

```